src/peer.rs

```diff
@@ -1,10 +1,92 @@
+extern crate serde;
+extern crate serde_json;
 use crate::event::event_hash::EventHash;
 use failure::Error;
+use std::sync::RwLock;
+use std::fs::File;
+use std::io::Read;
 
 pub type PeerId = Vec<u8>;
 
+#[derive(Serialize, Deserialize)]
+pub struct LachesisPeerStruct {
+    #[serde(rename="PubKeyHex")]
+    id: PeerId,
+    #[serde(rename="NetAddr")]
+    net_addr: String,
+    #[serde(skip)]
+    used: i64,
+    #[serde(skip)]
+    height: i64,
```

Comment:
`height: Option<u64>` might be better here, as you can support twice the amount of data. However, it will require one additional branch, so that's worth keep in mind also.